manim/cli/cfg/commands.py

```diff
@@ -1,13 +1,9 @@
-"""
-cfg_subcmd.py
-------------
+import os
```

Comment:
add a module docs string?

Reply:
Sure, I'll add one for the other subcommands as well.